extension/packagemanagerextension/nbextension/scripts.js

```diff
@@ -0,0 +1,146 @@
+
+define([
+    'jquery',
+    './views',
+    './api'
+], function ($, views, api) {
+    "use strict";
+
+    let packageview = {
+
+        /*
+        This function populates the sidebar each time it is opened.
+        */
+
+        load: async function (dir) {
+
+            sessionStorage.setItem("project", dir);
+
+            let info = await api.getinfo(dir);
+            let data = info.packages;
+
+            let output = views.installed(data);
+            $('#installed-packages').html(output);
+
+            output = views.toinstall(data, []);
+            $('#to-install').html(output);
+            if (output === "") {
+                $('#to-install-main').css("display", "none");
+            }
+
+            let selectedPackages = [];
+            $(views.selectinstalled(selectedPackages));
```

Comment:
it's not the first place where you call functions like this.. why are you calling it inside `$(..)`?